conans/client/packager.py

```diff
@@ -120,3 +112,18 @@ def _create_aux_files(install_folder, package_folder, conanfile, copy_info):
     # Create the digest for the package
     digest = FileTreeManifest.create(package_folder)
     digest.save(package_folder)
+
+
+def _report_files_from_manifest(output, package_folder):
+    digest = FileTreeManifest.load(package_folder)
+    copied_files = list(digest.files())
+    try:
+        copied_files.remove(CONANINFO)
+    finally:
```

Comment:
This intended to be a ``except``?

Reply:
good question. this could happen in an abnormal situation - the user could remove the conaninfo.txt from the package. but I think you're right, if conaninfo.txt is not there, it's because something really wrong has happened. I'll remove it.

Reply:
done!